hc/front/forms.py

```diff
@@ -5,6 +5,7 @@
 class NameTagsForm(forms.Form):
     name = forms.CharField(max_length=100, required=False)
```

Comment:
Have a  doc string when defining your classes to give a brief explanation of what the class does.